examples/analytics.py

```diff
@@ -25,8 +26,8 @@
 # load the advertiser account instance
 account = client.accounts(ACCOUNT_ID)
 
-# limit request count and grab the first 10 line items from Cursor
-line_items = list(account.line_items(None, count=10))[:10]
```

Comment:
I'm noticing a misuse of `count` parameter here. Since in this case,`Cursor` class will read all cursor until it's exhausted so limiting request count leads time-consuming work due to extra API calls (because default is 200).